CSS/CSS.sublime-syntax

```diff
@@ -125,7 +121,31 @@ contexts:
           scope: punctuation.definition.comment.css
           pop: true
 
-  custom-media:
+  at-charset:
+    - match: \s*((@)charset\b)\s*
+      captures:
+        1: keyword.control.at-rule.charset.css
+        2: punctuation.definition.keyword.css
+      push:
+        - meta_scope: meta.at-rule.charset.css
+        - include: at-rule-punctuation
+        - include: literal-string
+
+  # @counter-style
+  # https://drafts.csswg.org/css-counter-styles-3/#the-counter-style-rule
+  at-counter-style:
+    - match: \s*((@)counter-style\b)\s+(?:(?i:\b(decimal|none)\b)|({{ident}}))\s*(?={)
+      captures:
+        1: keyword.control.at-rule.counter-style.css
+        2: punctuation.definition.keyword.css
+        3: invalid.illegal.counter-style-name.css
+        4: entity.other.counter-style-name.css
```

Comment:
I'm inclined to say use an `entity.other` scope and not `entity.name`. The `entity.name` scopes (other than `entity.name.tag` and `entity.name.section`) are for types and data structures.

There will be some docs coming out with the next build that covers scope names some more. However, the recommendation is that color schemes highlight `entity.name` all the same except for the tag and section variations. Generally, these should all be in the symbol list and index also.

`entity.other`, however, is currently used for attribute names and class names used when dealing with inheritance. In a perfect world, I'd love to see the tag and section scopes moved into `entity.other`, but that would break pretty much every color scheme ever.